packages/graphql-yoga/__tests__/error-masking.spec.ts

```diff
@@ -186,15 +189,14 @@ describe('error masking', () => {
     })
     const body = JSON.parse(await response.text())
     expect(body).toMatchInlineSnapshot(`
-            Object {
-              "data": null,
```

Comment:
I think we should always have `data` even if it is `null`;
https://spec.graphql.org/draft/#sec-Response-Format

Reply:
[If the GraphQL response contains the data entry and it is null, then the server SHOULD reply with a 2xx status code and it is RECOMMENDED it replies with 200 status code.](https://graphql.github.io/graphql-over-http/draft/#sel-FANNNJCAACIN__I)

Reply:
>  If the request failed before execution, due to a syntax error, missing information, or validation error, this entry must not be present.
Also I found this one, which I think we should cover as well.